homeassistant/helpers/trigger_template_entity.py

```diff
@@ -121,6 +121,7 @@ def __init__(
         self._rendered = dict(self._static_rendered)
         self._parse_result = {CONF_AVAILABILITY}
         self._attr_device_class = config.get(CONF_DEVICE_CLASS)
+        self.error = False
```

Comment:
Should this really be public? Also, can we give it a more descriptive name?

Reply:
No, I renamed it